src/librbd/cache/pwl/LogMap.cc

```diff
@@ -250,7 +250,7 @@ LogMapEntries<T> LogMap<T>::find_map_entries_locked(const BlockExtent &block_ext
  * represent the entire block extent of the GenericWriteLogEntry, and the
  * WriteLogMapEntry is added to the set.
  *
- * The set must not contain overlapping WriteLogMapEntrys. WriteLogMapEntrys
+ * The set must not contain overlapping WriteLogMapEntries. WriteLogMapEntries
```

Comment:
Probably wrong. Perhaps
```suggestion
 * The set must not contain overlapping `WriteLogMapEntry`s. `WriteLogMapEntry`s
```

Reply:
Looks like our comments crossed: I suggested skipping the class name entirely in another thread.